apps/frontend/src/renderer/components/task-detail/SubtaskActionList.tsx

```
@@ -0,0 +1,489 @@
+        {showScore && score > 0 && (
+          <ScoreBadge score={score} scoreBreakdown={scoreBreakdown} />
+        )}
+        {hasDetail && (
+          <button
+            onClick={() => setIsExpanded(!isExpanded)}
+            className={cn(
+              'flex items-center gap-1 text-[10px] px-1.5 py-0.5 rounded shrink-0',
+              'text-muted-foreground hover:text-foreground hover:bg-secondary/50 transition-colors',
+              isExpanded && 'bg-secondary/50'
+            )}
+          >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)